### PR TITLE
Style Guide

### PR DESCRIPTION
**Pull request by cayouetten**
ESLint, mocha, DWA Style Guide implemmentation. README.md update

*console.log warning ignored for situational needs

### CODE REVIEW

**Comment by buchholz-sean** [comment]
Does this commit include all the changes? It looks like `package.json` doesn't currently include all the dependencies to automate the test.

**Review by buchholz-sean — CHANGES_REQUESTED** [review]
Unable to run automated linter test. `package.json` does not include required dependencies.

**Comment by cayouetten** [comment]
Updates implemented, please review

**Comment by buchholz-sean** [comment]
Ack! Your `test/__eslint.js` doesn't include the path to your `src` folder.

After changing, codebase fails strict test due to `no-console` warnings.

File: test/__eslint.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const lint = require('mocha-eslint');
+
+// Array of paths to lint
+// Note: a seperate Mocha test will be run for each path and each file which
+// matches a glob pattern
+const paths = [
+  'bot',
```
Comment (buchholz-sean) [inline test/__eslint.js]: Please make sure you include your app's path here so the test will hit it.

**Review by buchholz-sean — CHANGES_REQUESTED** [review]
Test path does not include app `src` folder.

**Comment by cayouetten** [comment]
Update, see commit for changes.

**Review by buchholz-sean — APPROVED** [review]
LGTM 👍 

Aside: Please make sure to commit .gitignore so installing modules to a cloned repo doesn't trigger file changes in Atom.

**Pull request merged**